spotinst/azure_v3/stateful_node_azure_login/fields_spotinst_stateful_node_azure_login.go

```diff
@@ -0,0 +1,107 @@
+package stateful_node_azure_login
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	azurev3 "github.com/spotinst/spotinst-sdk-go/service/stateful/providers/azure"
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+	"github.com/spotinst/terraform-provider-spotinst/spotinst/commons"
+)
+
+func Setup(fieldsMap map[commons.FieldName]*commons.GenericField) {
+
+	fieldsMap[Login] = commons.NewGenericField(
+		commons.StatefulNodeAzureLogin,
+		Login,
+		&schema.Schema{
+			Type:     schema.TypeList,
+			Optional: true,
```

Comment:
is it not required configuration? can we create stateful node without login configuration?